monitoring/README.md

```diff
@@ -73,23 +80,30 @@ sudo pip3 install requests
 ```
 
 
-### On the Pis
-1. Run the following script to initialize the pis, copy all needed files to them, and install and run the monitoring service on them:
+### Run On the VMs to deploy monitoring on Pis
+1. Run commands below manually to set up ip tables for pis to access internet. These commands are included in monitoring_piman.sh because this configuration will be gone after rebooting your VM.
+```bash
+sudo echo 1 > /proc/sys/net/ipv4/ip_forward
+sudo iptables -t nat -A POSTROUTING -o ens3 -j MASQUERADE
```

Comment:
can you explain what ens3 and ens4 are? it might be different on different machines.

Reply:
Thanks for reminding me.